pkg/security/resolvers/usersessions/resolver_linux.go

```diff
@@ -18,12 +18,36 @@ import (
 
 	manager "github.com/DataDog/ebpf-manager"
 
-	"github.com/DataDog/datadog-agent/pkg/security/config"
 	"github.com/DataDog/datadog-agent/pkg/security/probe/managerhelper"
 	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
+	"github.com/DataDog/datadog-agent/pkg/security/secl/model/usersession"
 	"github.com/DataDog/datadog-agent/pkg/security/seclog"
 )
 
+// UserSessionKey describes the key to a user session
+type UserSessionKey struct {
+	ID      uint64
+	Cursor  byte
+	Padding [7]byte
+}
+
+// UserSessionData stores user session context data retrieved from the kernel
+type UserSessionData struct {
+	SessionType usersession.Type `field:"-" json:"-"`
```

Comment:
Do we really need `field` stuff ? I think this is only for model right ?

Reply:
Yes, gonna remove it thanks